elements/campaign-brand-messaging/campaign-brand-messaging.js

```diff
@@ -0,0 +1,46 @@
+import {
+  registerElement,
+  BulbsHTMLElement,
+} from 'bulbs-elements/register';
+import { filterBadResponse, getResponseJSON } from 'bulbs-elements/util';
+import invariant from 'invariant';
+
+class CampaignProductShot extends BulbsHTMLElement {
+
+  handleRequestSuccess (data) {
+    if (!data.brand_messaging) {
+      console.log('Brand messaging not set on this campaign');
```

Comment:
I would prefer to not log this kind of thing. It should be required and throw an error using invariant.